convert2rhel/logger.py

```diff
@@ -58,11 +62,52 @@ class LogLevelFile:
     label = "DEBUG"
 
 
-def setup_logger_handler(log_name, log_dir):
+class FileBufferHandler(BufferingHandler):
+    name = "file_buffer_handler"
+
+    def __init__(self, capacity):
+        logging.handlers.BufferingHandler.__init__(self, capacity)
+        # the FileLogger handler that we are logging to
+        self._target = None
+
+    @property
+    def target(self):
+        """The computed target value is the FileHandler for logging, if none exists and we are flushing, be on the safe
+        side and return a nullhandler
+
+        :return logging.Handler: Either the file handler setup or a NullHandler
+        """
+        # We already have the filehandler, so we can
+        if self._target:
+            return self._target
+
+        for handler in logger.handlers:
+            if handler.name == "file_handler":
+                self._target = handler
+                return self._target
+        return None
```

Comment:
To comply with the docstring, should we change this to return 

```suggestion
        return logging.NullHandler()
```

Instead of returning None?

Reply:
Fixed!